Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM elasticsearch:2.4.0
+FROM elasticsearch:5.5.0
 
-RUN bin/plugin install lmenezes/elasticsearch-kopf/v2.1.2
-RUN bin/plugin install io.fabric8/elasticsearch-cloud-kubernetes/2.4.0_01
+RUN bin/elasticsearch-plugin install io.fabric8:elasticsearch-cloud-kubernetes:5.5.0
```

Comment:
can you upgrade to 5.5.1, now fabric8:elasticsearch-cloud-kubernetes plugin  is on 5.5.1. [see here](https://github.com/fabric8io/elasticsearch-cloud-kubernetes/releases)

Reply:
Now 5.5.2 is out

Reply:
https://github.com/jetstack/elasticsearch-pet/pull/4/commits/0834e484acd6c16f6332da70ba53da899d102060